### PR TITLE
Add clobber task to remove the output path directory

### PR DESCRIPTION
**Pull request by justin808**
Addresses #131.

### CODE REVIEW

**Comment by justin808** [comment]
Sample output
```
➜  ~/shakacode/react-on-rails/react_on_rails-with-webpacker (kaizencodes-develop) ✗ rake assets:clobber                                                                                      ✹ [23:07:54]
I, [2017-04-03T23:08:18.019257 #89542]  INFO -- : Removed /Users/justin/shakacode/react-on-rails/react_on_rails-with-webpacker/public/assets
Webpacker is installed 🎉 🍰
Using /Users/justin/shakacode/react-on-rails/react_on_rails-with-webpacker/config/webpack/paths.yml file for setting up webpack paths
Removed webpack output path directory /Users/justin/shakacode/react-on-rails/react_on_rails-with-webpacker/public/assets/webpack
➜  ~/shakacode/react-on-rails/react_on_rails-with-webpacker (kaizencodes-develop) ✗ rake webpacker:clobber                                                                                   ✹ [23:08:18]
Webpacker is installed 🎉 🍰
Using /Users/justin/shakacode/react-on-rails/react_on_rails-with-webpacker/config/webpack/paths.yml file for setting up webpack paths
Removed webpack output path directory /Users/justin/shakacode/react-on-rails/react_on_rails-with-webpacker/public/assets/webpack
```

File: lib/tasks/webpacker/clobber.rake [hunk 1]
```diff
@@ -0,0 +1,17 @@
+require "webpacker/configuration"
+
+namespace :webpacker do
+  desc "Remove the webpack compiled output directory"
+  task clobber: ["webpacker:verify_install", :environment] do
+    output_path = Webpacker::Configuration.output_path
+    FileUtils.rm_r(output_path) if File.exist?(output_path)
+    puts "Removed webpack output path directory #{output_path}"
```
Comment (dhh) [inline lib/tasks/webpacker/clobber.rake]: Don't think we need to be chatty if this is working. Don't think assets:clobber says something on success, does it?

Comment (justin808) [inline lib/tasks/webpacker/clobber.rake]: @dhh Please see the snippet above. assets:clobber

```
I, [2017-04-03T23:08:18.019257 #89542]  INFO -- : Removed /Users/justin/shakacode/react-on-rails/react_on_rails-with-webpacker/public/assets
```

**Comment by justin808** [comment]
Anything pending to merge this?

**Comment by dhh** [comment]
👍

**Pull request merged**